src/components/Analytics/Analytics.container.js

```diff
@@ -0,0 +1,384 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { injectIntl, intlShape } from 'react-intl';
+
+import AnalyticsComponent from './Analytics.component';
+import { logout } from '../Account/Login/Login.actions';
+import { getUser, isLogged } from '../App/App.selectors';
+import { showNotification } from '../Notifications/Notifications.actions';
+import API from '../../api';
+import messages from './Analytics.messages';
+
+export class AnalyticsContainer extends Component {
+  static propTypes = {
+    intl: intlShape.isRequired,
+    isLogged: PropTypes.bool.isRequired,
+    user: PropTypes.object.isRequired,
+    boards: PropTypes.array.isRequired,
+    logout: PropTypes.func.isRequired,
+    showNotification: PropTypes.func.isRequired
+  };
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      days: 30,
+      isFetching: false,
+      usage: {
+        max: 100,
+        min: 0,
+        data: Array.from(Array(30), () => 0)
+      },
+      totals: {
+        words: {
+          title: props.intl.formatMessage(messages.totalWords)
+        },
+        phrases: {
+          title: props.intl.formatMessage(messages.totalPhrases)
+        },
+        boards: {
+          title: props.intl.formatMessage(messages.boardsUsed)
+        },
+        editions: {
+          title: props.intl.formatMessage(messages.tilesEdited)
+        }
+      },
+      categoryTotals: {
+        navigation: {
+          value: 0,
+          title: props.intl.formatMessage(messages.navigationEvents)
+        },
+        speech: {
+          value: 0,
+          title: props.intl.formatMessage(messages.speechEvents)
+        },
+        edit: {
+          value: 0,
+          title: props.intl.formatMessage(messages.editingEvents)
+        }
+      },
+      topUsed: { symbols: [], boards: [] }
+    };
+  }
+
+  clientId = '';
+  timerId = '';
+
+  async componentDidMount() {
+    const { intl, showNotification } = this.props;
+    this.setState({ isFetching: true });
+    try {
+      this.clientId = await this.getGaClientId();
+      const totals = await this.getTotals(this.state.days);
+      const usage = await this.getUsage(this.state.days);
+      const categoryTotals = await this.getCategoryTotals(this.state.days);
+      const topUsed = this.getTopUsed(totals);
+      this.setState({ totals, categoryTotals, usage, topUsed, isFetching: false });
+    } catch (err) {
+      this.setState({ isFetching: false });
+      showNotification(intl.formatMessage(messages.loadingError));
+      console.log(err.message);
+    }
+  }
+
+  getSymbolSources() {
+    const { boards } = this.props;
+    const images = boards
```

Comment:
Nitpick: You can probably use [`flat`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/flat) instead of `reduce` here.